app/components/Form/ProjectFundingAgreementForm.tsx

```diff
@@ -91,6 +91,15 @@ const ProjectFundingAgreementForm: React.FC<Props> = (props) => {
         projectFormChange {
           formDataRecordId
         }
+        anticipatedFundingAmountPerFiscalYear {
+          edges {
+            # eslint-disable-next-line relay/unused-fields
```

Comment:
Not sure how to avoid this as the whole node needs to be passed via `formContext`

Reply:
Could this be turned into it's own fragment?